src/helpers/notifme.ts

```diff
@@ -331,15 +331,18 @@ export const sendNotification = async (message: string) => {
   if (getSecret("NOTIFICATION_TELEGRAM") && getSecret("NOTIFICATION_TELEGRAM_BOT_KEY")) {
     console.log("Sending Telegram");
     try {
-      await axios.post(
-        `https://api.telegram.org/bot${getSecret("NOTIFICATION_TELEGRAM_BOT_KEY")}/sendMessage`,
-        {
-          parse_mode: "Markdown",
-          disable_web_page_preview: true,
-          chat_id: getSecret("NOTIFICATION_TELEGRAM_CHAT_ID"),
-          text: message.replace(/_/g, '\\_'),
-        }
-      );
+      const chatIds = getSecret("NOTIFICATION_TELEGRAM_CHAT_ID").split(",");
```

Comment:
I wrote `NOTIFICATION_TELEGRAM_BOT_KEY` there by mistake.

It should be `NOTIFICATION_TELEGRAM_CHAT_ID`.

Sorry for that ><